pypeit/core/flux_calib.py

```diff
@@ -802,16 +803,21 @@ def fit_zeropoint(wave, Nlam_star, Nlam_ivar_star, gpm_star, std_dict,
     points spacing, which determines the scale of variation of the
     sensitivity function is determined by the nresln parameter.
 
+    wave, Nlam_star, Nlam_ivar_star, gpm_star, std_dict must either be:
```

Comment:
I am not following where this list functionality is occurring? It appears to me that it is not used anywhere, and I also don't see why it is necessary, since you can simply loop over the routine achieving the same things. Can you please clarify. It seems to me that if lists are passed in this routine will crash, but I probably missed something.